activemq-broker/src/main/java/org/apache/activemq/broker/region/DestinationStatistics.java

```diff
@@ -186,6 +177,11 @@ public void reset() {
             maxUncommittedExceededCount.reset();
             networkEnqueues.reset();
             networkDequeues.reset();
+
+            MessageFlowStatsImpl tmpMessageFlowStats = messageFlowStats;
```

Comment:
Nit: I usually like to make all of these final just to prevent any future mistakes